Automatic Validation ended with:
> 2025-04-30T11:41:30.0390837Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.9.0 File: Storj.Uplink.locale.en-US.yaml Manifest Error: The multi file manifest has inconsistent field values. [PackageVersion] Value: 1.125.2 File: Storj.Uplink.yaml Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.9.0 File: Storj.Uplink.yaml
 2025-04-30T11:41:30.0404738Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.9.0 File: Storj.Uplink.locale.en-US.yaml Manifest Error: The multi file manifest has inconsistent field values. [PackageVersion] Value: 1.125.2 File: Storj.Uplink.yaml Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.9.0 File: Storj.Uplink.yaml

(Automated response - build 1048.)

--- a/manifests/s/Storj/Uplink/1.127.1/Storj.Uplink.locale.en-US.yaml
+++ b/manifests/s/Storj/Uplink/1.127.1/Storj.Uplink.locale.en-US.yaml
@@ -1,3 +1,5 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
 PackageIdentifier: Storj.Uplink
 PackageVersion: 1.127.1
 PackageLocale: en-US

--- a/manifests/s/Storj/Uplink/1.127.1/Storj.Uplink.locale.en-US.yaml
+++ b/manifests/s/Storj/Uplink/1.127.1/Storj.Uplink.locale.en-US.yaml
@@ -105,4 +105,4 @@ ReleaseNotes: |-
   - 17d97aa storagenode/hashstore: use custom map type
 ReleaseNotesUrl: https://github.com/storj/storj/releases/tag/v1.125.2
 ManifestType: defaultLocale
-ManifestVersion: 1.9.0
+ManifestVersion: 1.10.0

--- a/manifests/s/Storj/Uplink/1.127.1/Storj.Uplink.yaml
+++ b/manifests/s/Storj/Uplink/1.127.1/Storj.Uplink.yaml
@@ -1,8 +1,8 @@
 # Created with WinGet Updater using komac v2.10.1
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
 
 PackageIdentifier: Storj.Uplink
-PackageVersion: 1.125.2
+PackageVersion: 1.127.1
 DefaultLocale: en-US
 ManifestType: version
-ManifestVersion: 1.9.0
+ManifestVersion: 1.10.0

